documentation/documentation/docusaurus.config.ts

```diff
@@ -0,0 +1,136 @@
+import {themes as prismThemes} from 'prism-react-renderer';
+import type {Config} from '@docusaurus/types';
+import type * as Preset from '@docusaurus/preset-classic';
+
+const config: Config = {
+  title: 'Project péristeronas',
+  tagline: 'Dinosaurs are cool',
```

Comment:
Should this be our tagline

Reply:
It would be fun tho xD

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/c2af2565e6508f3e8caa588f821307480a79c078